### ISSUE TITLE
Aarch64 executable  broken - missing Vgen::emit()

### ISSUE THREAD

**Issue by swalk-cavium** [facebook/hhvm#7951]
### HHVM Version

hphp/hhvm/hhvm --version
HipHop VM 3.20.0-dev (dbg)
Compiler: heads/experiment_080417-0-g5c04e03d8e8b1e7ec276887e1149daa6a4d96325
Repo schema: bad3993bbe3016938bd67808d17f6938334a50b8

/mnt/cdn/hhvm/hhvm$ git branch -v
* experiment_080417   5c04e03 Refined upper bound for array types and other collection types

### Expected result
Some fibonacci numbers

### Actual result

/mnt/cdn/hhvm/hhvm$ TRACE="printir:9,vasm:6" hphp/hhvm/hhvm -vEval.Jit=1 bench.php 

Assertion failure: /mnt/cdn/hhvm/hhvm/hphp/runtime/vm/jit/vasm-arm.cpp:242: void HPHP::jit::{anonymous}::Vgen::emit(const Inst&) [with Inst = HPHP::jit::cmpqim]: assertion `false' failed.
unimplemented instruction: cmpqim in B0

**Comment by swalk-cavium** [comment]
Executable still fails when you supply the missing emitters.

**Comment by vielmetti** [comment]
Same errors with this config:

```
HipHop VM 3.20.0-dev (rel)
Compiler: heads/master-0-gcc3c6f2764282cb5cdd59db580edfba098dec469
Repo schema: 8cd270a3e3b405d961878acb6f5a535295b950b1
```

```
$ git branch -v
* master cc3c6f2 Enable hot static code mapping onto huge pages
```

**Comment by swalk-cavium** [comment]
This issue was closed by a50c189 from PR#7956.  Pull request was accepted on Aug 18, 2017.
